op-chain-ops/genesis/db_migration.go

```diff
@@ -155,16 +204,19 @@ func MigrateDB(ldb ethdb.Database, config *DeployConfig, l1Block *types.Block, m
 		Difficulty:  common.Big0,
 		Number:      new(big.Int).Add(header.Number, common.Big1),
 		GasLimit:    (uint64)(config.L2GenesisBlockGasLimit),
-		GasUsed:     gasUsed,
+		GasUsed:     (uint64)(config.L2GenesisBlockGasLimit) * config.EIP1559Elasticity,
```

Comment:
Possible bug: The elasticity should only be used to compute the maximum limit based on the target. This multiplication doesn't make sense. And it's higher than the above `GasLimit`, which it should never be.

Reply:
Interesting. Maybe it's meant to be a division rather than a multiplication?

Reply:
If the transition block doesn't include any transactions, I think we should just set `GasUsed` to 0.

Reply:
Ah so the gas limit here is the limit and not the target. I was going for trying to have the eip1559 basefee computation be smooth. If we know that its going to mine a bunch of empty blocks at start then i'm fine with it just being set to 0 and save the complexity